lib/irc.js

```diff
@@ -217,11 +218,15 @@ function Client(server, nick, opt) {
                 // Random welcome crap, ignoring
                 break;
             case 'err_nicknameinuse':
-                if (typeof (self.opt.nickMod) == 'undefined')
-                    self.opt.nickMod = 0;
-                self.opt.nickMod++;
-                self.send('NICK', self.opt.nick + self.opt.nickMod);
-                self.nick = self.opt.nick + self.opt.nickMod;
+                if (typeof (self.opt.nickMod) == 'undefined' || typeof self.opt.nickMod == 'number') {
```

Comment:
:+1: Will do, IDK why I didn't in the first place. Must be because previous implementation wasn't